### PR TITLE
Update lab2-application-and-service.md

### PR DESCRIPTION
**Pull request by shalk**
修改prestart 为 postStart

**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by gaocegege** [comment]
Thanks for your contribution! :tada: :+1:

File: tutorials/lab2-application-and-service.md [hunk 1]
```diff
@@ -1529,7 +1529,7 @@ how.nice.to.look=fairlyNice
 4.  进一步学习 kubectl run。回答如何向 Pod 注入环境变量？如何查看是否注入成功？
 5.  进一步学习 kubectl rollout。回答如何通过 kubectl rollout 将应用回滚到指定版本？
 6.  Pod LivenessProbe 实验中，检查方式采用的是 http 模式。回答如何使用 exec 进行健康检查？请写出 yaml 文件。
-7.  进一步学习 Pod Lifecycle。回答如何使用 Prestart Hook？请写出 yaml 文件。
+7.  进一步学习 Pod Lifecycle。回答如何使用 postStart Hook？请写出 yaml 文件。
```
Comment (gaocegege) [inline tutorials/lab2-application-and-service.md]:
```diff
+ 7.  进一步学习 Pod Lifecycle。回答如何使用 PostStart Hook？请写出 yaml 文件。
- 7.  进一步学习 Pod Lifecycle。回答如何使用 postStart Hook？请写出 yaml 文件。
```

**Comment by bbbmj** [comment]
Thanks for your contribution! 🎉 👍

**Pull request merged**